api/v1/status_routes.go

```diff
@@ -98,8 +98,8 @@ func HandlePatchStatus(rw http.ResponseWriter, r *http.Request, p httprouter.Par
 			if status.VUs.Valid {
 				newConfig.VUs = status.VUs
 			}
-			if updateErr := executor.UpdateConfig(r.Context(), newConfig); err != nil {
-				apiError(rw, "Config update error", updateErr.Error(), http.StatusInternalServerError)
+			if updateErr := executor.UpdateConfig(r.Context(), newConfig); updateErr != nil {
+				apiError(rw, "Config update error", updateErr.Error(), http.StatusBadRequest)
```

Comment:
@MStoykov We need that `errorcheck` linter ;)

Also note the change to a 400 response code here, which is what was expected by the test, and will probably make sense in most cases.